web/themes/custom/usagov/templates/views-view-list--sub-children.html.twig

```diff
@@ -20,75 +20,94 @@
 
-<div class="usagov-cards margin-top-3">
-  <ul class="usa-card-group">
-    {% for row in rows %}
-    <li class="tablet:grid-col-4 usa-card">
-      <a href="{{ drupal_url('node/' ~ row.content["#row"].nid) }}" class="usa-card__container usagov-card text-no-underline">
-      {% if term_name == "Life Events Landing Page" %}
-       <div class="life-events-landing-page-card-image" style="background-image: url({{file_url(row.content["#row"]._entity.field_navigation_banner_image.entity.field_media_image.entity.fileuri)}})"></div>
-       <div class="life-events-landing-page-card-title">{{row.content["#row"].node_field_data_title}}</div>
-      {% else %}
-        {{- row.content -}}
-      {% endif %}
-      </a>
-    </li>
-    {% endfor %}
-  </ul>
-</div>
-{% if lang == "en" %}
-<p><span class="text-bold">Looking for something else?</span> <a id="something-else" href="/#all_topics">Explore all topics and services</a></p>
-{% else %}
-<p><span class="text-bold">¿Busca algo más? <a id="something-else" href="/es#all_topics">Encuentre más temas y servicios</a></span></p>
-{% endif %}
+	<div class="usagov-cards margin-top-3">
+		<ul class="usa-card-group">
+			{% for row in rows %}
+				<li class="tablet:grid-col-4 usa-card">
+					<a href="{{ drupal_url('node/' ~ row.content["#row"].nid) }}" class="usa-card__container usagov-card text-no-underline">
+						{% if term_name == "Life Events Landing Page" %}
+							<div class="life-events-landing-page-card-image" style="background-image: url({{file_url(row.content["#row"]._entity.field_navigation_banner_image.entity.field_media_image.entity.fileuri)}})"></div>
+							<div class="life-events-landing-page-card-title">{{row.content["#row"].node_field_data_title}}</div>
+						{% else %}
+							{{- row.content -}}
+						{% endif %}
+					</a>
+				</li>
+			{% endfor %}
+		</ul>
+	</div>
+	{% if lang == "en" %}
+		<p>
+			<span class="text-bold">Looking for something else?</span>
+			<a id="something-else" href="/#all_topics">Explore all topics and services</a>
+		</p>
+	{% else %}
+		<p>
+			<span class="text-bold">¿Busca algo más?
+				<a id="something-else" href="/es#all_topics">Encuentre más temas y servicios</a>
+			</span>
+		</p>
+	{% endif %}
+
+{% elseif term_name == "Home Page" %}
 
-{% elseif term_name == "Standard Page" %}
-{# standard_page #}
+	{% if view.current_display == "block_4" %}
+		{% include directory ~ '/templates/includes/homepage-cards.html.twig' %}
+	{% endif %}
 {% endif %}
 
-{% if view.current_display == "block_4" %}
-  {# block_4 is the intro_page display for this view which is used on the homepage #}
- <span id="all_topics"></span>
-  <div class="usagov-cards">
-    <ul class="usa-card-group">
-      {% for row in rows %}
+{% if view.current_display == "block_4" and term_name != "Home Page"%}
+
+	{# block_4 is the intro_page display for this view which is used on the homepage #}
```

Comment:
Am I correct in thinking that this clause is for backward compatibility and that we'll be able to remove it after the new Home Page has gone live?

Reply:
> I'm seeing this message in my Drupal recent log messages whenever I visit my home page:
> 
> > User notice: Could not load image style max_1920w
> 
> I also see near-identical messages for "max_1280w" and "max_960w"
> 
> (There's more to the message, but my first question is -- do you see this too?)

I did but I hadn't seen that before. I had incorrectly named those 3 image styles. I pushed the name fixes up!

Reply:
Yes and no. This did allow for the cards to show before going through the steps above but truly I just forgot to delete it before opening the PR. Deleted now.